dynamic_security_analysis.sh

```diff
@@ -0,0 +1,261 @@
+#!/bin/bash
```

Comment:
This project's convention is for scripts to be in the `/scripts` directory -- please move this there.